server/graphql/v1/mutations.js

```diff
@@ -208,6 +209,18 @@ const mutations = {
       return updateUserEmail(remoteUser, email);
     },
   },
+  setChangelogViewDate: {
```

Comment:
Any reason to use API v1 for this? All new features should usually be on API v2.

Reply:
@znarf : My thinking was that all the user related mutations are on v1. I thought it's best to keep this with that until we migrate all the v1 mutations to v2 related to the user. But I am okay with adding this mutation to v2 as well. 🤔

Reply:
Thank you for sharing the rationale.

I definitely advice to not introduce it in GraphQL v1 and properly implement it in GraphQL v2.

Reply:
@znarf : Yep, just moved it to GraphQL v2. 😄